lib/active_merchant/billing/gateways/mercado_pago.rb

```diff
@@ -61,6 +61,10 @@ def verify(credit_card, options = {})
         end
       end
 
+      def inquire(authorization, options = {})
```

Comment:
This introduces a different method signature than what @almalee24 established in 430664e58bf630a13244ca855be71eccfd35444c. My thinking is that this `(authorization, options)` signature is more consistent with other public transaction action methods. Also, gateways across the gem are most likely to support a `GET by authorization id` request, if not both, and are already used to seeing `order_id` in options. Thoughts?

Reply:
I agree. Can you update the method signature in the `dlocal` gateway as part of this commit

Reply:
Updated!